examples/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "../tsconfig.json",
+  "compilerOptions": {
+    "allowSyntheticDefaultImports": true,
+    "strict": true,
+    "module": "esnext",
+    "noEmit": true
+  }
+}

```

Comment:
CRA auto adds the compilerOptions here, but I am extending from the above `tsconfig.json` to still maintain the same rules above.